### PR TITLE
Fix rid choice setup stress

### LINKED ISSUE #22510: [x64] Failed to locate the downloaded library
**Job:**
https://ci.dot.net/job/dotnet_coreclr/job/master/view/x64/job/jitstress/job/x64_checked_ubuntu_gcstress0xc_jitstress1_tst/104/

**Log:**
```
Generating MSBuild file /mnt/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_ubuntu_gcstress0xc_jitstress1_tst/tests/src/Common/stress_dependencies/obj/stress_dependencies.csproj.nuget.g.props.
  Generating MSBuild file /mnt/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_ubuntu_gcstress0xc_jitstress1_tst/tests/src/Common/stress_dependencies/obj/stress_dependencies.csproj.nuget.g.targets.
  Restore completed in 2.98 sec for /mnt/j/workspace/dotnet_coreclr/master/jitstress/x64_checked_ubuntu_gcstress0xc_jitstress1_tst/tests/src/Common/stress_dependencies/stress_dependencies.csproj.
Failed to locate the downloaded library
Failed to set up stress dependencies.
```

### PR DESCRIPTION
**Pull request by jashook**
Fixes #22510

### CODE REVIEW

File: tests/setup-stress-dependencies.sh [hunk 1]
```diff
@@ -101,23 +96,81 @@ if [ ! -e $libInstallDir ]; then
...
+        __HostOS=Linux
+        ;;
+esac
+
+isPortable=0
+
+source "${scriptDir}"/../init-distro-rid.sh
+initDistroRidGlobal ${__BuildOS} x64 ${isPortable}
+
+# Hack, replace the rid to ubuntu.14.04 which has a valid non-portable
```
Comment (echesakovMSFT) [inline tests/setup-stress-dependencies.sh]: Do we plan to eventually remove this hack?

Comment (jashook) [reply tests/setup-stress-dependencies.sh]: Yes

**Comment by jashook** [comment]
Merged as this file is unused by ci, and I do not want to waste resources.

**Comment by sdmaclea** [comment]
Thanks

**Pull request merged**